src/lib/y2network/autoinst_profile/interface_section.rb

```diff
@@ -163,7 +163,7 @@ def self.attributes
       #  @return [String] ???
 
       # @!attribute bonding_slaveX
-      #  @return [String] bonding slave on position X
+      #  @return [String] bonding port on position X
```

Comment:
So what about AY profiles? Whatever we do, document it.
- commit messages
- PR description
- AY docs? There are some now: https://documentation.suse.com/sles/15-SP3/single-html/SLES-autoyast/#CreateProfile-Network-Interfaces

Reply:
for now I'd keep it. If anything is going to be done in AY profiles it will has to be in separate commit as it will require quite some work with respect to backward compatibility.